sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -350,19 +340,19 @@
             "in": "header",
             "required": false,
             "type": "string",
-            "description": "The continuation token to get the next page of results."
+            "description": "The continuation token used to get the next page of results."
           },
           {
             "name": "x-ms-max-item-count",
             "in": "header",
             "required": false,
             "type": "string",
```

Comment:
I thought we combined all our suggested changes into this swagger, not just description changes?

Reply:
It is one swagger but I was not making any of those changes in the current PR so that it is easy to review. I will do a full sweep of those and send another PR.

Reply:
That's fair. We can just add another task to follow up on it then.